test_unstructured/partition/test_api.py

```diff
@@ -73,34 +73,6 @@ def test_partition_via_api_from_filename(monkeypatch):
     assert elements[0].metadata.filetype == "message/rfc822"
 
 
-def test_partition_via_api_custom_url(monkeypatch):
-    """
-    Assert that we can specify api_url and requests are sent to the right place
-    """
```

Comment:
*this test is now failing with the updated SDK client since it is raising an error about not having an api_key. When you define an api_key, another error is still raised